ap-plus/config/_colour.scss

```diff
@@ -0,0 +1,16 @@
+// Black
+$black: #000;
+
+// White
+$white: #fff;
+
+// Red
+$red: #b8202e; // APW $red (used for error styles)
+
+// Maroon
```

Comment:
I'd add all the ones they've given us (so maroon 1 – 4) then we've got them to use – and not keen on using AP red, can we use the maroon instead?